app/jobs/govbox/submit_message_draft_job.rb

```diff
@@ -26,13 +27,18 @@ def perform(message_draft, schedule_sync: true, upvs_client: UpvsEnvironment.upv
       objects: build_objects(message_draft)
     }.compact
 
-    sktalk_api = upvs_client.api(message_draft.thread.box).sktalk
+    sktalk_api = upvs_client.api(box).sktalk
     success, response_status, response_body = sktalk_api.receive_and_save_to_outbox(message_draft_data)
 
+    box.message_submission_requests.create(
```

Comment:
Takto asi nebudeme vediet rozlisit ci to bolo poslane hromadne alebo nie, ze?

Reply:
Nie, chceme to rozlisovat?

Reply:
No podla mna ano, hromadne akcie su imho ta pridana hodnota, cize tie by mali byt nejako zalogovane. Mozno na urovni akcii toto drzat?

Reply:
Pridany `bulk` atribut.